privacyidea/lib/eventhandler/base.py

```diff
@@ -447,6 +456,15 @@ def check_condition(self, options):
             if not bool(m):
                 return False
 
+        if CONDITION.COUNTERNAME in conditions:
+            # Can be countername==1000
+            if not compare_generic_condition(conditions.get(CONDITION.COUNTERNAME),
+                                             counter_read,
```

Comment:
We should check if the counter exists, otherwise we will get an error when comparing `None` with some value.

Reply:
If the counter does not exist, the condition would not be true.
However, the request should not fail with an exception.

Reply:
According to the new commit 8ce52d54 it works as expected.

Reply:
Darn. I am wrong in regards to Python 3.